client/wfnews-war/src/main/angular/src/app/app.module.ts

```diff
@@ -49,6 +49,10 @@ import {
 } from "@wf1/wfdm-document-management-api";
```

Comment:
What are we using from WFDM on this page currently?
Make sure to clean up unused imports (we can always bring them back if we add components again)